src/jit/flowgraph.cpp

```diff
@@ -542,27 +542,17 @@ bool Compiler::fgBlockContainsStatementBounded(BasicBlock*  block,
 //    node      - The node to be inserted.
 //
 // Return Value:
-//    Returns the new (potentially) GT_STMT node.
+//    Returns the new (potentially) statement.
 //
 // Notes:
 //    If 'stmt' is not already a statement, a new statement is created from it.
 //    We always insert phi statements at the beginning.
 //    In other cases, if there are any phi assignments and/or an assignment of
 //    the GT_CATCH_ARG, we insert after those.
 
-GenTreeStmt* Compiler::fgInsertStmtAtBeg(BasicBlock* block, GenTree* node)
+GenTreeStmt* Compiler::fgInsertStmtAtBeg(BasicBlock* block, GenTreeStmt* stmt)
```

Comment:
Perhaps the return type should be changed to void? It's not very useful to return what was passed in as parameter.

Reply:
All functions `GenTreeStmt* fgInsertStmtSomewhere` return an argument as a result, will change it later in a separate PR, because for some of them we have users:
`afterStmt = fgInsertStmtAfter(block, afterStmt, newStmt);` 

It came from the time when these functions accepted trees and could return a stmt created to box that tree.